L1Trigger/Phase2L1Taus/plugins/HPSPFTauProducer.cc

```diff
@@ -220,7 +220,7 @@ void HPSPFTauProducer::fillDescriptions(edm::ConfigurationDescriptions& descript
   desc.add<double>("minSeedJetPt", 30.0);
   desc.add<double>("maxChargedRelIso", 1.0);
   desc.add<double>("minSeedChargedPFCandPt", 5.0);
-  desc.add<edm::InputTag>("srcL1PFCands", edm::InputTag("l1ctLayer1", "PF"));
+  desc.add<edm::InputTag>("srcL1PFCands", edm::InputTag("l1tLayer1", "PF"));
```

Comment:
Is this a bug, or the bug is in the master instead?

Reply:
It seems this line will be overwrite by
https://github.com/cms-sw/cmssw/blob/master/L1Trigger/Phase2L1Taus/python/HPSPFTauProducerPF_cfi.py#L5
`srcL1PFCands = "l1tLayer1:PF",`

However, let's @cms-sw/l1-l2 confirms.

Reply:
Agree. Let's spend a little time to clean up things. 

In addition, I see few places in master that still use `l1ctLayer1` which I assume them to change to `l1tLayer1`
https://github.com/cms-sw/cmssw/search?q=l1ctLayer1

Reply:
This PR is correct. I will make a bug-fix PR to master.

Reply:
Thanks @cecilecaillol

Reply:
Will the fixes for [L1Trigger/Phase2L1ParticleFlow/plugins/DeregionizerProducer.cc](https://github.com/cms-sw/cmssw/pull/39704/files#diff-188e0d47aea503e77d5e3a6c5f20cf6e1624a4fe386bf786c35016e8aebbe3fb) and L1Trigger/Phase2L1ParticleFlow/test/make_l1ctLayer1_dumpFiles_cfg.py come in separated backport PR?

Reply:
Yes: https://github.com/cms-sw/cmssw/pull/39705